frontend/www/js/omegaup/arena/contest_practice.ts

```diff
@@ -8,6 +8,11 @@ import arena_ContestPractice, {
   ActiveProblem,
 } from '../components/arena/ContestPractice.vue';
 
+export interface ClarificationRequest {
+  problemAlias: string;
+  message?: string;
```

Comment:
```suggestion
  message: string;
```

perdón >< ya es lo último.

Reply:
Listo